modules/flatgeobuf/docs/api-reference/flatgeobuf-loader.md

```diff
@@ -0,0 +1,75 @@
+# FlatGeobufLoader
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.3-blue.svg?style=flat-square" alt="From-v2.3" />
+</p>
+
+Loader for the [FlatGeobuf](https://bjornharrtell.github.io/flatgeobuf/) format, a binary FlatBuffers-encoded format that defines geospatial geometries.
+
+| Loader         | Characteristic                                            |
+| -------------- | --------------------------------------------------------- |
+| File Extension | `.fgb`,                                                   |
+| File Type      | Binary                                                    |
+| File Format    | [FlatGeobuf](https://bjornharrtell.github.io/flatgeobuf/) |
+| Data Format    | [Geometry](/docs/specifications/category-gis)             |
+| Supported APIs | `load`, `parse`, `parseSync`                              |
+
+## Usage
+
+```js
+import {FlatGeobufLoader} from '@loaders.gl/flatgeobuf';
+import {load} from '@loaders.gl/core';
+
+// GeoJSON objects containing local coordinates decoded from tile origin to a range of [0 - (bufferSize / tileExtent), 1 + (bufferSize / tileExtent)]
+const geometryData = await load(url, MVTLoader);
+
+// Array containing GeoJSON Features
+const loaderOptions = {
+  mvt: {
+    coordinates: 'wgs84',
+    tileIndex: {
+      x: 0,
+      y: 0,
+      z: 0
+    }
+  }
+};
+
+const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
```

Comment:
search replace MVT with case insensitive settings.

Reply:
Yes, as you've noticed I put docs off till the rest of the PR is ready.